client/stylesheets/_toCat.scss

```diff
@@ -5110,7 +5111,12 @@ On click the images will be displayed at normal size to complete the effect
 /* View Feedback */
 
 .profileUserFeedbackView {
-    width: 60%;
+    width: 90%;
+}
+
+.profileUserFeedbackView h2 {
```

Comment:
Merge rule `.profileUserFeedbackView h2` with rule on line 5113<br>Selector `profileUserFeedbackView` should be written in lowercase with hyphens